packages/remix-node/magicExports/remix.ts

```diff
@@ -13,4 +13,4 @@ export {
   unstable_parseMultipartFormData,
 } from "@remix-run/node";
 
-export type { UploadHandler, UploadHandlerArgs } from "@remix-run/node";
+export type { UploadHandler, UploadHandlerPart } from "@remix-run/node";
```

Comment:
We should also keep `UploadHandlerArgs` here, as it would otherwise be a breaking change

```suggestion
export type { UploadHandler, UploadHandlerArgs, UploadHandlerPart } from "@remix-run/node";
```

Reply:
It's a breaking change anyways as the props on the type have changed. Upload handlers are unstable so this should be fine.